specification/elastic/resource-manager/Microsoft.Elastic/stable/2023-06-01/elastic.json

```diff
@@ -0,0 +1,2510 @@
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/UpgradableVersionsList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "UpgradableVersions_Details": {
+            "$ref": "./examples/UpgradableVersions_Details.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/upgrade": {
+      "post": {
+        "tags": [
+          "DeploymentUpdate"
+        ],
+        "summary": "Upgradable version for a monitor resource.",
+        "operationId": "Monitor_Upgrade",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Elastic Monitor Upgrade Parameters",
+            "schema": {
+              "$ref": "#/definitions/ElasticMonitorUpgrade"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Monitor_Upgrade": {
+            "$ref": "./examples/Monitor_Upgrade.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/listAllTrafficFilters": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Get the list of all traffic filters for the account.",
+        "operationId": "AllTrafficFilters_list",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Pagination Support?

Reply:
@visingla-ms 
As discussed offline, we're just creating a stable version for the preview API versions which are already available and no new changes are being pushed here.
PFB the previous PRs:
https://github.com/Azure/azure-rest-api-specs/pull/22638
https://github.com/Azure/azure-rest-api-specs/pull/23343
https://github.com/Azure/azure-rest-api-specs/pull/23602
https://github.com/Azure/azure-rest-api-specs/pull/23650
https://github.com/Azure/azure-rest-api-specs/pull/24031